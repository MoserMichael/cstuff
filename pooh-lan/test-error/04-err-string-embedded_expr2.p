

a = ' aaa [ foo() ] bbb [ gooo() ] aa '


oyoyoy()

a = ' aaa [ foo() ] bbb [ gooo() ] aa '
