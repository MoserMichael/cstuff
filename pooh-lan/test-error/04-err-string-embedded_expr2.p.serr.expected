
test-error/04-err-string-embedded_expr2.p(3): Error: the function foo is not defined, can't call a function that is not defined
      3 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |a = ' aaa [ foo() ] bbb [ gooo() ] aa '
        |............^^^

test-error/04-err-string-embedded_expr2.p(3): Error: the function gooo is not defined, can't call a function that is not defined
      3 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |a = ' aaa [ foo() ] bbb [ gooo() ] aa '
        |..........................^^^^

test-error/04-err-string-embedded_expr2.p(6): Error: the function oyoyoy is not defined, can't call a function that is not defined
      6 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |oyoyoy()
        |^^^^^^

test-error/04-err-string-embedded_expr2.p(8): Error: the function foo is not defined, can't call a function that is not defined
      8 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |a = ' aaa [ foo() ] bbb [ gooo() ] aa '
        |............^^^

test-error/04-err-string-embedded_expr2.p(8): Error: the function gooo is not defined, can't call a function that is not defined
      8 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |a = ' aaa [ foo() ] bbb [ gooo() ] aa '
        |..........................^^^^
Type checking failed
